src/common/contracts/lido-locator/lido-locator.service.ts

```diff
@@ -0,0 +1,15 @@
+import { LidoLocator, LIDO_LOCATOR_CONTRACT_TOKEN } from '@lido-nestjs/contracts';
+import { Injectable, Inject } from '@nestjs/common';
+import { ExecutionProvider } from 'common/execution-provider';
+
+@Injectable()
+export class LidoLocatorService {
+  constructor(
+    @Inject(LIDO_LOCATOR_CONTRACT_TOKEN) protected readonly contract: LidoLocator,
+    protected readonly provider: ExecutionProvider,
+  ) {}
+
+  async getStakingRouter(blockTag: number | string) {
+    return await this.contract.stakingRouter({ blockTag });
```

Comment:
https://eips.ethereum.org/EIPS/eip-1898

```{ blockTag: { blackhash: '0x...' }}```

pls test it with alchemy

Reply:
fixed